iceberg/iceberg-handler/src/main/java/org/apache/iceberg/mr/hive/HiveIcebergStorageHandler.java

```diff
@@ -737,6 +746,22 @@ public void executeOperation(org.apache.hadoop.hive.ql.metadata.Table hmsTable,
     }
   }
 
+  private static void expireSnapshotByIds(Table icebergTable, String[] idsToExpire,
+      ExecutorService deleteExecutorService) {
+    if (idsToExpire.length != 0) {
+      ExpireSnapshots expireSnapshots = icebergTable.expireSnapshots();
+      for (String id : idsToExpire) {
```

Comment:
Maybe just a question. If we have multiple snapshotId, what's will happen when one of snapshotId does not exist?

e.g. We have three snapshots, A, B and C, but B dose not exist acutually. In this case, will the snapshot A and B be expired sucessfully?

Reply:
it will be successful, you wanted to get rid of a snapshot, it isn't there already, so we are already sorted, it would expire the other ones & return